crates/nu-command/Cargo.toml

```diff
@@ -87,17 +87,18 @@ serde_yaml = "0.9.4"
 sha2 = "0.10.0"
 sqlparser = { version = "0.32.0", features = ["serde"], optional = true }
 sysinfo = "0.29"
-tabled = "0.12.0"
-terminal_size = "0.2.1"
-titlecase = "2.0.0"
-toml = "0.7.1"
-unicode-segmentation = "1.10.0"
-ureq = { version = "2.6.2", default-features = false, features = [ "json", "charset", "native-tls", "gzip", ] }
-native-tls = "0.2.11"
-url = "2.2.1"
-uuid = { version = "1.3.0", features = ["v4"] }
-wax = { version = "0.5.0" }
-which = { version = "4.4.0", optional = true }
+tabled = { version = "0.12.2", features = ["color"], default-features = false }
+terminal_size = "0.2"
+titlecase = "2.0"
+toml = "0.7"
+unicode-segmentation = "1.10"
+ureq = { version = "2.6", default-features = false, features = [ "json", "charset", "native-tls", "gzip", ] }
+native-tls = "0.2"
+url = "2.2"
+uuid = { version = "1.3", features = ["v4"] }
+wax = { version = "0.5" }
+which = { version = "4.4", optional = true }
```

Comment:
@atahabaki 
i'm not sure why all these versions are changing here :thinking: 
according to `git blame`, this hunk comes from 3ba1613e8, a.k.a. *str-expand: resolve conflicts a bit*, but this commit changes the versions rather than just fix conflicts, right?

maybe there is an explaination for all these version changes :yum:

Reply:
I couldn't remember the git command to remove Cargo.lock and Cargo.toml from my branch history. I don't have explaination for those changes. I just wanted "bracoxide v0.1.0", damn it XDXD i screwed up.

Reply:
@atahabaki 
simpler, just paste the following block in `Cargo.toml`
```toml
tabled = "0.12.0"
terminal_size = "0.2.1"
titlecase = "2.0.0"
toml = "0.7.1"
unicode-segmentation = "1.10.0"
ureq = { version = "2.6.2", default-features = false, features = [ "json", "charset", "native-tls", "gzip", ] }
native-tls = "0.2.11"
url = "2.2.1"
uuid = { version = "1.3.0", features = ["v4"] }
wax = { version = "0.5.0" }
which = { version = "4.4.0", optional = true }
```
and the dependencies should be as before :ok_hand:

Reply:
noice, fixed in ...

the global diff on `crates/nu-command/Cargo.toml` is now only [this](https://github.com/nushell/nushell/pull/9290/files#diff-6b6659f5abe38891ef52d15c14998d251080fdc7b2900813d1bcbda82590584d) :ok_hand: